File: plugins/OrchidSwagger/src/main/java/com/eden/orchid/languages/SwaggerModule.java
package com.eden.orchid.languages;

import com.eden.orchid.api.registration.OrchidModule;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;
import com.eden.orchid.api.theme.components.OrchidComponent;

public class SwaggerModule extends OrchidModule {

    @Override
    protected void configure() {
        addToSet(OrchidComponent.class,
                SwaggerComponent.class);

        addToSet(PluginResourceSource.class,
                SwaggerResourceSource.class);
    }
}

File: plugins/OrchidSwagger/src/main/kotlin/com/eden/orchid/languages/SwaggerModule.kt
package com.eden.orchid.languages

import com.eden.orchid.api.registration.OrchidModule
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource
import com.eden.orchid.api.theme.components.OrchidComponent
import com.eden.orchid.languages.components.SwaggerComponent

class SwaggerModule : OrchidModule() {

    override fun configure() {
        addToSet(OrchidComponent::class.java,
                SwaggerComponent::class.java)

        addToSet(PluginResourceSource::class.java,
                SwaggerResourceSource::class.java)
    }

}
